### PR TITLE
Azure DevOps Migration

### LINKED ISSUE #9: Migrate from AppVeyor and Travis to Azure DevOps

### PR DESCRIPTION
**Pull request by ctolkien**
Fixes #9 

- [ ] Resolve versioning scenario

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+      "version": "2.2.300"
```
Comment (poke) [inline global.json]: This might break with the ASP.NET Core 3.x PR. You should just install a 3.0 SDK and build 2.1/2.2/3.0 with it.

Comment (ctolkien) [reply global.json]: Tests failed when running under the 3.0 SDK...

Actually that might have been unrelated. Let me try again

Comment (ctolkien) [reply global.json]: Ended up installing both SDKs.
